src/renderers/WebGLCubeRenderTarget.js

```diff
@@ -103,12 +100,34 @@ WebGLCubeRenderTarget.prototype.fromEquirectangularTexture = function ( renderer
 
 	material.uniforms.tEquirect.value = texture;
 
-	const mesh = new Mesh( new BoxBufferGeometry( 5, 5, 5 ), material );
+	const mesh = new Mesh( geometry, material );
 
-	scene.add( mesh );
+	//
 
 	const camera = new CubeCamera( 1, 10, this );
-	camera.update( renderer, scene );
+	camera.updateMatrixWorld();
+
+	const currentRenderTarget = renderer.getRenderTarget();
+	const generateMipmaps = this.texture.generateMipmaps;
+
+	this.texture.generateMipmaps = false;
```

Comment:
Doing this means will always be `false` and thus using `LinearMipmapLinearFilter` is invalid. Is this intended?

This might break existing code that use to generate the background like so:
```js
var options = {
	generateMipmaps: true,
	minFilter: THREE.LinearMipmapLinearFilter,
	magFilter: THREE.LinearFilter
};

scene.background = new THREE.WebGLCubeRenderTarget( 1024, options ).fromEquirectangularTexture( renderer, texture );
```

Reply:
For this particular case.... I don't think we need mipmaps for the background? 🤔